ReleaseNotes.md

```diff
@@ -11,6 +11,14 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+
+### 05/12/2021 -
+##### Version microsoft/oms:ciprod00512021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod00512021 (linux)
+##### Version microsoft/oms:win-ciprod05122021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod05122021 (windows)
```

Comment:
please go-ahead and remove the windows agent image as we only doing linux agent release.